lib/stack.js

```diff
@@ -1,19 +1,25 @@
+//first in last out
+const LinkedList = require("../lib/linked-list.js");
+
+
 class Stack {
     constructor() {
-        // this.store = ...
-        throw new Error("This method has not been implemented!");
+        this.store = new LinkedList();
     }
 
-    push() {
-        throw new Error("This method has not been implemented!");
+    push(element) {
+        return this.store.addLast(element);
     }
 
     pop() {
-        throw new Error("This method has not been implemented!");
+        const removeItem = this.store.getLast();
+        this.store.delete(removeItem);
+        return removeItem;
+
     }
 
     isEmpty() {
-        throw new Error("This method has not been implemented!");
+        return this.store.length() === 0;
     }
 
     toString() {
```

Comment:
✨ This works! LinkedList also has a `__str__` method you could take advantage of!